app/src/main/res/values-de/strings.xml

```diff
@@ -6,7 +6,7 @@
     <string name="action_settings">Einstellungen</string>
     <string name="action_logout">Abmelden</string>
     <string name="action_search">Suchen</string>
-    <string name="action_updates">Aktualisierungen</string>
+    <string name="action_updates">Updates</string>
```

Comment:
I was unsure about this.
My thought process was like this:
It's already called "Apps". "Aktualisierung" sounds very clunky. And "Update" is a commonly used word, at least in my experience.
My main point is that it should be consistent. Before my changes it was very mixed.